src/main/java/frc/robot/RobotContainer.java

```diff
@@ -52,10 +52,10 @@ public RobotContainer() {
         // Turning is controlled by the X axis of the right stick.
         new RunCommand(
             () -> m_robotDrive.drive(
```

Comment:
I think you need to change this to use the new rate-limiting function rather than the existing drive method. Double check the logic that REV has because that should be the source of truth.